readme.md

```diff
@@ -38,7 +42,8 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
-
+SQL databases are relational databases; NoSQL DBs are non-relational databases. You could use either in many situations but when you have many complex
+relationships between datasets, it's more efficient to use a SQL database.
 ```
```

Comment:
Also, documents in a NoSQL DB collection do not need to follow the exact same format.